contracts/SimpleYellowToken.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
+
+import '@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol';
+
+/**
+ * @notice Yellow and Canary utility token is a simple ERC20 token with permit functionality.
+ * All the supply is minted to the deployer.
+ */
+contract SimpleYellowToken is ERC20Permit {
+	/// @dev Token maximum supply
+	uint256 public immutable TOKEN_SUPPLY_CAP;
+
+	/**
+	 * @dev Simple constructor, passing arguments to ERC20Permit and ERC20 constructors.
+	 * Mints the supply cap to the deployer.
+	 * @param name Name of the Token.
+	 * @param symbol Symbol of the Token.
+	 * @param supplyCap Maximum supply of the Token.
+	 */
+	constructor(
+		string memory name,
+		string memory symbol,
+		uint256 supplyCap
```

Comment:
Name this supply or tokenSupply

Reply:
ef857e48c551976dc091766f37676b89e005f14f